Application/Client/src/app/createProfile/tpls/createProfile.html

```diff
@@ -3,89 +3,170 @@
     <ol class="breadcrumb col-lg-11">
       <li><a>Home</a></li>
       <li><a>my profile</a></li>
-      <li class="active">edit info</li>
+      <li class="active">{{userProfileData.UserName}}</li>
```

Comment:
back end team  is using PASACAL naming convention so we have used this variable as it is on front end side